x-pack/plugins/rule_registry/server/routes/get_browser_fields_by_feature_id.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from '@kbn/core/server';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import { schema } from '@kbn/config-schema';
+
+import { RacRequestHandlerContext } from '../types';
+import { BASE_RAC_ALERTS_API_PATH } from '../../common/constants';
+
+export const getBrowserFieldsByFeatureId = (router: IRouter<RacRequestHandlerContext>) => {
+  router.get(
+    {
+      path: `${BASE_RAC_ALERTS_API_PATH}/browser_fields`,
+      validate: {
+        query: schema.object({
+          featureIds: schema.string({ minLength: 1 }),
+        }),
+      },
+      options: {
+        tags: ['access:rac'],
+      },
+    },
+    async (context, request, response) => {
+      try {
+        const racContext = await context.rac;
+        const alertsClient = await racContext.getAlertsClient();
+
+        const { featureIds } = request.query;
+        const indices = await alertsClient.getAuthorizedAlertsIndices(featureIds.split(','));
+
+        if (!indices) {
+          return response.notFound({
+            body: {
+              message: `indices for featureIds [${featureIds}] not found`,
+              attributes: { success: false },
+            },
+          });
+        }
```

Comment:
can we move this logic for 011yIndices? and say that for now we are only dealing with o11y alerts.